lms/static/js/jquery.gradebook.js

```diff
@@ -27,49 +21,38 @@ var Gradebook = function($element) {
 
 	var moveDrag = function(e) {
 		var offset = e.pageX - mouseOrigin;
-		var targetLeft = clamp(tableOrigin + offset, -maxScroll, 0);
+		var targetLeft = clamp(tableOrigin + offset, maxScroll, 0);
 
 		updateHorizontalPosition(targetLeft);
-
-		setShadows(targetLeft);
 	};
 
-	var stopDrag = function(e) {
+	var stopDrag = function() {
 		$body.css('-webkit-user-select', 'auto');
 		$body.unbind('mousemove', moveDrag);
 		$body.unbind('mouseup', stopDrag);
 	};
 
-	var setShadows = function(left) {
-		var padding = 30;
-
-		var leftPercent = clamp(-left / padding, 0, 1);
-		$leftShadow.css('opacity', leftPercent);
-	
-		var rightPercent = clamp((maxScroll + left) / padding, 0, 1);
-		$rightShadow.css('opacity', rightPercent);
-	};
-
 	var clamp = function(val, min, max) {
-	    if(val > max) return max;
-	    if(val < min) return min;
-	    return val;
+		if(val > max) {
+			return max;
```

Comment:
Use spaces, not tabs. I suggest that you configure your editor to do this automatically.